azerothcore/azerothcore-wotlk
# fix(DB/Creature) Add CTM to 4 flying creatures

## Changes Proposed:
-  ~~Adds creature_template_movement entries to Gahz'ranka (15114) and Zulian Crocolisk (15043) from ZG.~~
-  Adds CTM to Outlands flying creatures 15242, 18707, 224411 so that they won't fall to the ground.
-  Adds CTM to Hinterlands creature 8276 so that they fly properly.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/12183
- Closes https://github.com/chromiecraft/chromiecraft/issues/3662
- Closes part of https://github.com/azerothcore/azerothcore-wotlk/issues/11814 as it still needs sniffs for the waypoints.

## SOURCE:


## Tests Performed:

- Tested ingame
- 


## How to Test the Changes:


1. `.go c id 15242`, `.go c id 18707`, `.go c id 22441` `.go c id 8276`
2.  See if all creatures fly properly and don't fall to the ground. Note: Teribus still needs waypoints, so he'll be static.
3. ~~`.additem 19975 5` `.additem 19974` `.go xyz -11695 -1770 13 309`~~
4. ~~Use lure to summon Gahz'ranka if needed. Crocolisks are nearby. I don't really know how you'd test these two since we don't have the water Evade issues much anymore.~~

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
